src/styles/index.scss

```diff
@@ -2,2 +1,209 @@
+@import './utils/variables';
+@import './blocks/page';
+@import './blocks/nav';
+@import './blocks/header';
+@import './blocks/stars';
+@import './blocks/buy';
+@import './blocks/card';
+
+// Variaveis
+
+$main-accent: rgba(6, 11, 53, 1);
+$secondary: rgba(97, 96, 112, 1);
+$white: rgba(255, 255, 255, 1);
+$blue-accent: rgba(0, 172, 220, 1);
+$gray-elements: rgba(243, 243, 243, 1);
+$transition-duration: 300ms;
+$gray-border: 1px solid $gray-elements;
+$title-hover: #34568b;
+$blue-border: 1px solid $blue-accent;
+$rounding: 5px;
+
+.buy {
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 12px 70px;
+  color: rgba(255, 255, 255, 1);
+  background-color: $blue-accent;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  border-radius: $rounding;
+  transition: $transition-duration;
+
+  &:hover {
+    background-color: $white;
+    color: $blue-accent;
+    border: $blue-border;
+  }
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 200px));
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  border: $gray-border;
+  padding: 32px 16px 16px;
+  border-radius: $rounding;
+  transition: $transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The 'transform: scale(1.2)' on hover for the '.card' class may cause layout shifts and affect user experience. Consider using a smaller scale factor or other visual effects that do not significantly alter the layout.